pkgs/by-name/se/seabios/package.nix

```
@@ -1,11 +1,37 @@
-{ lib
-, stdenv
-, fetchgit
-, acpica-tools
-, python3
-, writeText
+{
+  lib,
+  acpica-tools,
+  fetchgit,
+  python3,
+  stdenv,
+  writeText,
+  # Configurable options
+  ___build-type ? "csm",
```

```suggestion
  buildType ? "csm",
```
I don't see a reason for the underline-prefixed, kebab-case name.

---

Paranoia:

https://github.com/NixOS/nixpkgs/pull/324199#issuecomment-2212422318

there is a small, non-negligible chance of a package having the name `buildType`.

---

Fascinating. I actually got a similar thing when I was working on `starsector`. Leave it as is then.